main_tagging.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/google/go-github/v65/github"
+	"golang.org/x/oauth2"
+)
+
+func main() {
+	ctx := context.Background()
+	client := newGitHubClient(ctx)
+	org := os.Getenv("GITHUB_ORG")
+
+	keywordTopics := getKeyWords()
+
+	// COMMENT THIS TO AVOID UPDATING ALL REPOS WHILE TESTING
+	allRepos := getAllRepositories(ctx, client, org)
+	for _, repo := range allRepos {
+		repoName := *repo.Name
+		//
+
+		// TEST REPOS ONLY
+		// testRepos := []string{"tf-azurerm-module_reference-key_vault", "tf-aws-module_primitive-firehose_delivery_stream"}
+		// for _, repoName := range testRepos {
+		//
+
+		topics := getCurrentTopics(ctx, client, org, repoName)
+
+		topics, updated := updateCurrentTopics(topics, keywordTopics, repoName)
+
+		applyNewTopics(ctx, updated, client, org, repoName, topics)
+
+	}
+}
+
+func newGitHubClient(ctx context.Context) GitHubClient {
+	token := os.Getenv("GITHUB_TOKEN")
```

Comment:
I think the semantics for `os.Getenv` return an empty string if the environment variable is not set, right?

We might want to add some messaging here where if we get an empty string back, we exit with a nonzero code and an error message instructing the user to set the variable.